### ISSUE TITLE
Windows 8 and driver signature enforcement

### ISSUE THREAD

**Issue by nc54** [mlt/schwinn810#25]
I had a computer meltdown and ended up installing windows 8. I am running into problems installing the driver for windows 8.   The modified driver will not install because it was never meant to work on Win8.   So I downloaded their new win8 compatible driver and modified  `slabvcp.inf`  with 
`%USB\VID_10C4&PID_EA61.DeviceDesc%=silabser.Dev, USB\VID_10C4&PID_EA61`
under both  `[SiLabs]` as well as `[NTamd64]`. This gets a bit further into the install but ultimately fails due to the hash not matching.     I have tried updating the driver as well as installing from the EXE and both ways fail.  Any clue how to fix this?

![screenshot](https://dl.dropbox.com/u/19832238/Hash.PNG)

**Comment by mlt** [comment]
Looks like MS tries hard to limit freedom not only with [restricted boot](http://www.fsf.org/campaigns/secure-boot-vs-restricted-boot) but also by reenforcing driver integrity.

Quick googling shows [this](http://www.pcpro.co.uk/blogs/2012/08/06/getting-older-drivers-to-work-in-windows-8/). There should be a [way to enable installation of unsigned drivers](http://laslow.net/2012/03/14/disable-driver-signature-enforcement-in-windows-8/).

**Comment by nc54** [comment]
I followed the instructions in the second link with success.   I modified the new driver from the silicon labs website so it would recognize the bridge and commented out the line about the catalog.  Then used advanced startup options and selected option 7 to install unsigned drivers.  You may want to update the driver in the downloads.  Consider this issue resolved, thanks to your mad googling skills.

**Comment by mlt** [comment]
Great!

>  You may want to update the driver in the downloads.

Github deprecated **downloads** section :(

Was inf editing much different from the former version? If so, could you provide a patch that needs to be applied to original inf file? Or just put original and updated in dropbox.

**Comment by nc54** [comment]
Same patch as you did, just comment out the line with "catalog"  Here is
the link to the modified inf
https://dl.dropbox.com/u/19832238/slabvcp.inf

With download gone, where do we download updates?  I never installed the one with that deletes tracks from the watch and was going to do that now.

**Comment by mlt** [comment]
> With download gone, where do we download updates?

That is still an open question. I though that someone might have a place to host everything. Though after looking at all your dropbox links... perhaps 20 GB/day is quite a limit. I guess it worth still creating some mailing list for notifications on updates then. But I did not decide where yet.

I'd say it should be manageable to install python with few extra packages and then just use Tortoise Git to pull updates.

> I never installed the one with that deletes tracks from the watch and was going to do that now.

I think I did not end up making corresponding binary.
